financial-connections/src/main/java/com/stripe/android/financialconnections/ui/theme/Color.kt

```diff
@@ -61,3 +76,73 @@ internal data class FinancialConnectionsColors(
     val iconSuccess: Color,
     val iconAttention: Color
 )
+
+@Immutable
+internal data class FinancialConnectionsV3Colors(
+    val textDefault: Color,
+    val textSubdued: Color,
+    val textDisabled: Color,
+    val textWhite: Color,
+    val textBrand: Color,
+    val textCritical: Color,
+    val iconDefault: Color,
+    val iconSubdued: Color,
+    val iconWhite: Color,
+    val iconBrand: Color,
+    val buttonPrimary: Color,
+    val buttonPrimaryHover: Color,
+    val buttonPrimaryPressed: Color,
+    val buttonSecondary: Color,
+    val buttonSecondaryHover: Color,
```

Comment:
What does this refer to, given the absence of hovering on mobile?

Reply:
For now just brought the entire color palette from this [Figma](https://www.figma.com/file/wXQ8NJApqSJiLmxxANDRTs/%E2%9C%A8-Connections-3.0?type=design&node-id=4887%3A218476&mode=dev), need to verify if it'll be used on some instances on mobile (tap ripples, etc)